main/src/main/java/org/apache/james/jdkim/api/Result.java

```diff
@@ -112,7 +112,7 @@ private String getHeaderText(boolean withReason) {
                     reasonMsg = "valid signature";
                     break;
                 case NONE:
-                    reasonMsg = "not signed";
+                    reasonMsg = "unknown error";
```

Comment:
I am not familiar enough with jdkim API to be fully assertive but from reading the spec there should be 2 levels of results 
- one for the entire message
- one for each signatures

For a signature verification  result,  type `NONE` doesn't make any sense since it is defined as being for unsigned messages in the specification 
At the message level (aggregate) however, none is not an error, it states that a DKIM authentication was attempted but no signature was found. 

The class constructor always takes a signatureRecord so in a correct implementation this branch should probably be unreachable

Reply:
I left that NONE (could be UNKNOWN), for the case an exception is thrown that is not TempFailException or PermFailException ([here](https://github.com/apache/james-jdkim/blob/11e7505e6b56af4fe989756a4c3e5c8c9b2eb1d2/main/src/main/java/org/apache/james/jdkim/DKIMVerifier.java#L449)). While only TempFailException and PermFailException are throw, the NONE will be never used. The NONE from Authentication-Results doesn't makes sense for an individual signature verification, a dkim=none can be used if a message doesn't have a signature, but it's not the case here.